configs/components/runtime.rb

```diff
@@ -3,8 +3,13 @@
 
   pkg.add_source "file://resources/files/runtime/runtime.sh"
   if platform.name =~ /solaris-10/
-    pkg.build_requires "http://pl-build-tools.delivery.puppetlabs.net/solaris/10/pl-gcc-4.8.2-1.#{platform.architecture}.pkg.gz"
-    pkg.build_requires "http://pl-build-tools.delivery.puppetlabs.net/solaris/10/pl-binutils-2.25.#{platform.architecture}.pkg.gz"
+    if platform.architecture == 'sparc'
```

Comment:
These were the combinations of builds that worked. The latest pl-gcc and pl-binutils didn't work for Solaris i386. @ScottGarman 

P.S. I think this is the right combination, waiting on the sparc build to finish. I'll remove this line if it passes.